### ISSUE TITLE
Add tools for streamlining sample sizing

### ISSUE THREAD

**Issue by dzeber** [mozilla/mozanalysis#205]
This will integrate the tools for streamlining sample size computations prototyped in [this notebook](https://colab.research.google.com/drive/1XYEHXVLUF5bQJVyFIgI7BJoBG72kKugJ) in `frequentist_stats/sample_size.py`.

**Comment by m-d-bowerman** [comment]
I agree on adding the pretty-printing to an output class. There's a bit of difference in output between the various sample size calculation methods, so I'm not sure how easily we can combine them into an output class.
